templates/miq-template-ext-db.yaml

```diff
@@ -433,18 +433,167 @@ objects:
 
       <VirtualHost *:80>
         KeepAlive on
+        # Without ServerName mod_auth_mellon compares against http:// and not https:// from the IdP
+        ServerName https://%{REQUEST_HOST}
+
         ProxyPreserveHost on
         ProxyPass        /ws/ ws://${NAME}/ws/
         ProxyPassReverse /ws/ ws://${NAME}/ws/
-        ProxyPass        / http://${NAME}/
+
+        # For httpd, some ErrorDocuments must by served by the front-end httpd
```

Comment:
I think front-end is overloaded here. I think "httpd pod" would be better to distinguish it from the miq-frontend pod